### PR TITLE
DVX-697 Add support for assuming roles to package-toolkit..runtime.Utils getInputFiles

### PR DESCRIPTION
**Pull request by ErnestoLoma**

### CODE REVIEW

File: package-toolkit/runtime/src/main/kotlin/com/atlan/pkg/Utils.kt [hunk 1]
```diff
@@ -635,6 +635,12 @@ object Utils {
                         getInputFiles(sync, outputDirectory, prefix)
                     }
 
+                    "iam", "role" -> {
```
Comment (cmgrote) [inline package-toolkit/runtime/src/main/kotlin/com/atlan/pkg/Utils.kt:638]: Do these have to be just `iam` and `role`? Other object stores (besides S3) also support IAM and role-based auth — so if we at some point need to extend this in the future to support the same in those other cloud providers, just using `iam` or `role` is not going to be sufficiently distinct...

**Review by cmgrote — CHANGES_REQUESTED** [review]
I'm a bit concerned about the very generic labels `iam` and `role` being uniquely mapped to S3 (only)... But I'm guessing it'll be a bigger change on the underlying connector configmap to change these to something else (?)

If so, I'm OK moving ahead with this as-is for now, but let's at least put into the comments of the `Utils` class that this `iam` and `role` condition is specific to a particular connector configmap (i.e. the name of the connector configmap it's for).

**Review by cmgrote — APPROVED** [review]
Added comment for now, approving and merging...

File: package-toolkit/runtime/src/main/kotlin/com/atlan/pkg/Utils.kt (hunk 1 above)
Comment (ErnestoLoma) [reply package-toolkit/runtime/src/main/kotlin/com/atlan/pkg/Utils.kt:639]: @cmgrote That's what is in the Credentials file in `/tmp/credentials/success/result-0.json`. I don't have any control over that. I'd been more concerned with trying to change the configmap.

**Pull request merged**